recipes/arb-bio/6.1.rev16223/abs_path_fix.patch

```diff
@@ -0,0 +1,753 @@
+Index: BINDINGS/PERL/Makefile
+===================================================================
+--- BINDINGS/PERL/Makefile	(revision 16223)
++++ BINDINGS/PERL/Makefile	(working copy)
```

Comment:
Not needed. I tend to agree with your first conclusion but this is not relevant for this PR. Thanks! Other than the pinning everything looks good.